extensions/tasks/NLUCleanV0/index.ts

```diff
@@ -0,0 +1,86 @@
+import * as tl from 'azure-pipelines-task-lib/task';
+import * as tr from 'azure-pipelines-task-lib/toolrunner';
+import * as path from 'path';
+
+async function run() {
+    try {
+        const tool = await getDotNetNLUToolRunner();
```

Comment:
Try to figure out a way to share the `getDotNetNLUToolRunner` logic (e.g., with lerna)